<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPGADialog</class>
 <widget class="QDialog" name="FPGADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_connect">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>12</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_Message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>256</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_Message">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>42</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Send">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>72</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_State">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>102</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Device State:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_decevie_state">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>124</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>..........</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_decevie_speed">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>168</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>..........</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Speed">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>146</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>41</width>
     <height>17</height>
    </size>
   </property>
   <property name="text">
    <string>FPGA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>minh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
